bindings/c/test/apitester/run_c_api_tests.py

```diff
@@ -111,6 +118,8 @@ def parse_args(argv):
                         help='The timeout in seconds for running each individual test. (default 300)')
     parser.add_argument('--logging-level', type=str, default='INFO',
                         choices=['ERROR', 'WARNING', 'INFO', 'DEBUG'], help='Specifies the level of detail in the tester output (default=\'INFO\').')
+    parser.add_argument('--bg-enabled', type=str, default=None,
```

Comment:
--bg-enabled sound like a flag argument, better call it --bg-local-file-path or similar